source/less/base/variables.less

```
@@ -0,0 +1,3 @@
+@tablet-width: 768px;
+@desktop-width: 1440px;
```

Учитывая, что контейнер у нас 1280px, то и переключение можно сделать уже на нем.

```suggestion
@desktop-width: 1280px;
```